compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -542,6 +542,22 @@ impl<'a: 'ast, 'ast> LateResolutionVisitor<'a, '_, 'ast> {
                 err.span_label(base_span, fallback_label);
             }
         }
+        if let Some(err_code) = &err.code {
+            if err_code == &rustc_errors::error_code!(E0425) {
+                for label_rib in &self.label_ribs {
+                    for (label_ident, _) in &label_rib.bindings {
+                        if format!("'{}", ident) == label_ident.to_string() {
+                            let msg = format!(
+                                "a label with a similar name exists: `{}`",
```

Comment:
I think it would be better to suggest `missing ' to be used for label` rather than a label with similar ident. Maybe

> missing ' required for label: \`'{}'`

The message still doesn't seemed that good for me.

Reply:
i think the error message is fine as is. I would prefer a `span_suggestion` here, as that should make this even clearer.

Reply:
I use `span_suggestion` instead :+1: